src/org/zaproxy/zap/extension/formHandler/ZapAddOn.xml

```diff
@@ -0,0 +1,20 @@
+<zapaddon>
+	<name>Form Handler</name>
+	<version>1</version>
+	<status>alpha</status>
+	<description>This is a test of the form handling information box --ALPHA--</description>
```

Comment:
The "alpha" could be removed, the add-ons have a status (also, avoids the need to change other places when the add-on is promoted).